.github/workflows/analysis-lint.yml

```diff
@@ -21,7 +21,9 @@ jobs:
         run: gofmt -s -l -d .
 
       - name: Run golangci-lint
-        uses: reviewdog/action-golangci-lint@v2
+        uses: reviewdog/action-golangci-lint@v2.7.2
         env:
           REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with: { reporter: github-pr-review }
+        with:
+          reporter: github-pr-review
+          golangci_lint_version: v1.64.8
```

Comment:
I think this action was updated recently and its completely broken. It last ran 5 days ago on these versions [here](https://github.com/spacelift-io/terraform-provider-spacelift/actions/runs/13972230685/job/39116757699). I pinned it to the previously working version so it doesnt turn this PR into something its not.

Reply:
golangci-lint v2 was released a few days ago, it might have broken it